frontend/src/App.tsx

```diff
@@ -1,9 +1,38 @@
 import "./App.css";
+import { useState } from "react";
+
+import Input from "./components/Input";
+import TodoList from "./components/TodoList";
+import { Todo } from "./models/todo.model";
 
 function App() {
+  const [todos, setTodos] = useState<Todo[]>([
+    { description: "Hello world", id: "244454", isDone: false },
+    { description: "Hello Pakistan", id: "24454", isDone: false },
+    { description: "Hello GB", id: "24433", isDone: false },
+  ]);
+
+  let value: string = "";
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    console.log(event.target.value);
+  };
+
   return (
-    <div>
-      <p>Hello world!</p>
+    <div className="App">
+      <div className=" card">
+        <div className="flex-container">
+          <Input
+            onChange={onChange}
+            value={value}
+            placeholder="ENTER TASK"
```

Comment:
The `placeholder` attribute takes a value that is an example of the data a user can enter in the input field. It's the label that tells the name of the field.

For example, if I am taking an input for an email address, the placeholder I would use is `someone@example.com`.

Update the placeholder to something like "Learn React" (without the quotes).